### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ADefWebserver**
Better directions for using Azure OpenAI

### CODE REVIEW

**Comment by ADefWebserver** [comment]
It took me 2 hours to figure all this out and get the correct settings. Just wanted to help others :)

File: README.md [hunk 1]
```diff
@@ -147,9 +147,17 @@ To setup the client to use your deployment, you'll need to pass in `OpenAIClient
...
+var settings = new OpenAIClientSettings(resourceName: "your-resource", deploymentId: "your-deployment-id", apiVersion: "your-api-version");
 var api = new OpenAIClient(auth, settings);
 ```
+For example, when in the Azure Playground, you can click the `View Code` button and view a URL like this:
+
+https://myresource.openai.azure.com/openai/deployments/GPT-35-Turbo/chat/completions?api-version=2023-03-15-preview
+
+That would be translated as follows:
+`resourceName`: myresource
+`deploymentId`: GPT-35-Turbo
```
Comment (StephenHodgson) [inline README.md:159]: Do the deployment ids really correspond to the available model?

Comment (ADefWebserver) [reply README.md:159]: This is what I *just-got-working* on my project 😎

File: README.md [hunk 2]
```diff
@@ -147,9 +147,17 @@ To setup the client to use your deployment, you'll need to pass in `OpenAIClient
 
 ```csharp
 var auth = new OpenAIAuthentication("sk-apiKey");
-var settings = new OpenAIClientSettings(resourceName: "your-resource", deploymentId: "your-deployment-id");
+var settings = new OpenAIClientSettings(resourceName: "your-resource", deploymentId: "your-deployment-id", apiVersion: "your-api-version");
 var api = new OpenAIClient(auth, settings);
 ```
+For example, when in the Azure Playground, you can click the `View Code` button and view a URL like this:
+
+https://myresource.openai.azure.com/openai/deployments/GPT-35-Turbo/chat/completions?api-version=2023-03-15-preview
```
Comment (StephenHodgson) [inline README.md:155]:
```suggestion
```
https://{resourceName}.openai.azure.com/openai/deployments/{deploymentId}/chat/completions?api-version={apiVersion}
```
```

Comment (StephenHodgson) [reply README.md]: Weird I guess it doesn't like nested blocks

File: README.md [hunk 3]
```diff
@@ -147,9 +147,16 @@ To setup the client to use your deployment, you'll need to pass in `OpenAIClient
 
 ```csharp
 var auth = new OpenAIAuthentication("sk-apiKey");
-var settings = new OpenAIClientSettings(resourceName: "your-resource", deploymentId: "your-deployment-id");
+var settings = new OpenAIClientSettings(resourceName: "your-resource", deploymentId: "your-deployment-id", apiVersion: "your-api-version");
 var api = new OpenAIClient(auth, settings);
 ```
+For example, when in the Azure Playground, you can click the `View Code` button and view a URL like this:
+
```
Comment (StephenHodgson) [inline README.md:154]:
```suggestion
`https://{resourceName}.openai.azure.com/openai/deployments/{deploymentId}/chat/completions?api-version={apiVersion}`
```

File: README.md (hunk 1 above)
Comment (ADefWebserver) [reply README.md]: Yeah *you* create that `deploymentId` when you create a deployment. This is a *very* Microsoft thing that doesn't exist on OpenAI

**Review by StephenHodgson — APPROVED** [review]

**Pull request merged**